### PR TITLE
Kestrel Update + Details

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork )**

## Summary
kestrelO
kestrel

kestrelO
kestrel

The Bactrian is the big elephant in the room that I tackled, so here I am tackling the big bird in the room. This PR is focused on an updated model (_NOT a remodel from scratch_) for the Tarazed Kestrel Warship. I am aware that this ship is a daunting endeavor to touch as it is, so I am accepting the possibility that none of these changes go through. With that said:

The Kestrel is in one of those weird places. The model and shape itself, based off of the iconic Kestrel of the EV series, has it's own adored history and look behind it, and there aren't really any issues to be had with it. However, the turret mounts and gun mounts have definitely aged a bit along with other ES ships before reworks, and with reworks adding a bit more detail into the game, I've touched up the base model. 

Similar to other Tarazed ships, and notably the Blackbird (below is a comparison), I've given the Kestrel subtle engravings along the fuselage to give it slight more detail. Although this won't be entirely visible from zoomed out, the same could be said for the Blackbird, and it adds that nice little spice of detail to help round it out more as a structured ship design. 

blackbird
kestrel

Needless to say, I've updated the Kestrel turrets to the Tarazed mounts that are used on the Osprey and Falcon, and moved the lower ones in a location that makes more sense. Originally those two were shoved inside almost upside down in the hull sides, but now they are jutting outward and make more sense from a top-view perspective (and look good in the thumb as well). I've also given the guns a new look that is a bit beyond a cylinder with extrusions.

All-in-all, it's another love tap to a design to help age it more finely. 

The Pandora Box I'm avoiding on opening is the possibility of this affecting MZ's personal Kestrel ship. Like the previous ship touchups I've done, I want to avoid touching any pers ships unless explicitly told to do so. Regarding this, I have re-introduced the original Kestrel design and pointed the person file to it so that MZ's ship sprite remains the original. 

MZ

-------------------------------------------------------------------------------------------------------------------------------------------

Now, here comes the debatable part. I am mostly interested in the above being considered for addition, however, I have worked on a few goodies here as well. I've given the 3 (4 if you count the base) different variations of the Kestrel slight but noticeable differences in design to match the player's given choices, using a few design marks of Kestrels from the past, and adding bits that might make more sense. 

**The ordering of the following from left to right is: Original Model, Updated Base Model, Engine-focused Model, Shield & Armor-focused Model, and Weapon-focused model**

KestrelComparison

The Engine Model now features more detail around the engines, intricate/noticeable intakes, removal of the canards, and removal of some bulk near them. Most notably, I have added a second smaller pair of engines, with smaller scaled flares (which can be removed if not desired, as has been before). This gives the Kestrel a more sleeker, thinner appearance that accompanied it in Escape Velocity, while changing the rear to more engine bulk to reflect the players choice.

The Shield/Armor Model now features more bulk added around the fuselage (similar to what is already on it), and an added 4th plating on the wings, which is mostly self-explanatory, pertaining to the defense upgrade option. This give it a bit more of a beefy appearance.

The Weapon Model now features more reinforced hardpoints- the wing plates are swapped out for more intentional structures that are in-line with the gun hardpoints, and the turret mounts are accentuated slightly more. Most notably, however, is the _visible_ addition of 6 more gun mounts (which I needn't say much about how much the community loves that many guns).  As a little shout-out to Nova's Kestrel, I've added fins on the near edge of the wings as a small detail. 

While I'd love for all of these to be great representations for each version, I do know there is an argument to be had about visualization having an effect on which model the player chooses. My conclusive comment to that is that for new players, they are more choosing for the stats of the ship, and for return players, it may be both or either or the stats or the looks, which I think isn't a bad thing at all. 

Once again, though, I know that editing each version specifically may not be appropriate to some people, so if there is any debate to this (be it supportive or against), please mention so down in the comments. If this comes down to a fairly fast "unlikely", I will remove the variant portion of this PR, and it will simply be for the base model.

## Save File
This save file can be used to play through the new mission content:
[Sherrol Mane Kestrel.txt](https://github.com/endless-sky/endless-sky/files/8093645/Sherrol.Mane.Kestrel.txt)
If you have one, the model should reflect which version you chose. 

## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I uploaded the necessary image, blend, and texture assets here: https://drive.google.com/drive/folders/1ie2yMZvMSr0byx_gvKJvXWUS4sVdDZvJ?usp=sharing
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: https://github.com/endless-sky/endless-sky-high-dpi/pull/212

### CODE REVIEW

**Comment by WarlordMike** [comment]
She looks beautiful now, Saugia! 😍

I know she'll have to get a full work-over at some point, but these graphics would make for a fantastic stopgap until then.

**Comment by Saugia** [comment]
> She looks beautiful now, Saugia! 😍
> 
> I know she'll have to get a full work-over at some point, but these graphics would make for a fantastic stopgap until then.

Thanks Mike! As such an important ship in the game history and tied with MZ himself, I didn't want to take it too far, and similarly to other ship reworks, I wanted to keep the design close to the original. If we ever get enough people saying it needs a total makeover, I'd love to attempt the full thing as well, but for now to be safe this is what I've got going.

**Comment by Zitchas** [comment]
They look really good - although I think they could stand to have a little more obvious differentiation between them. Not much, but a little more. Especially the detailing around the engines for the engine one seems particularly hard to see.

File: data/human/kestrel.txt [hunk 1]
```diff
@@ -329,32 +327,42 @@ ship "Kestrel"
 
 
 ship "Kestrel" "Kestrel (More Engines)"
+	sprite "ship/kestrelE"
```
Comment (Amazinite) [inline data/human/kestrel.txt:330]: I'd follow the Marauder image naming scheme where the suffix isn't capitalized. That is, make the following changes:
* kestrelE to kestrele
* kestrelS to kestrels
* kestrelW to kestrelw
* kestrelO to kestrelo

**Review by Amazinite — CHANGES_REQUESTED** [review]
The PR description mentions that you've changed MZ's person definition to use the original image, but no such change appears to have been made. Also, any copyright update, or do you decline to claim copyright for the updated images (i.e. the copyright stays with MZ)?

File: data/human/kestrel.txt [hunk 2]
```diff
@@ -306,20 +306,18 @@ ship "Kestrel"
 		"Orca Plasma Thruster"
 		"Orca Plasma Steering"
 		"Hyperdrive"
-	engine -14 177
-	engine 14 177
-	bay "Fighter" -46 106 under
-	bay "Fighter" 46 106 under
```
[the remark this answers is not in the document]
Comment (Saugia) [reply data/human/kestrel.txt]: I was hungry.

File: data/human/kestrel.txt (hunk 1 above)
Comment (Saugia) [reply data/human/kestrel.txt]: Fixed. Changed kestrelo to kestreloriginal for more clarification also.

**Pull request merged**